config/cmake/HDF5PluginCache.cmake

```diff
@@ -0,0 +1,28 @@
+# This is the CMakeCache file.
```

Comment:
These are the plugin project settings, which translate from the hdf5 library build.